browser/css/btns.css

```diff
@@ -1,4 +1,4 @@
-.jsdialog-container .ui-button-box button,
+ffont.jsdialog-container .ui-button-box button,
```

Comment:
```suggestion
.jsdialog-container .ui-button-box button,
```

That is a typo I guess?

Reply:
thanks a lot